components/infieldprogresscircle/stories/template.js

```diff
@@ -0,0 +1,18 @@
+import { Template as ProgressCircle } from "@spectrum-css/progresscircle/stories/template.js";
+import "../index.css";
+
+export const Template = ({
+	customClasses = [],
+	rootClass,
```

Comment:
I also wonder if `spectrum-InfieldProgressCircle` should actually be here, instead of within the custom classes of progress circle.